Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is the second line.
```

Comment:
Something

---
In reply to: [59789954](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r59789954) [](ancestors = 59789954,59789925)